frontend/src/components/UserTables/LocationsTable.tsx

```diff
@@ -10,73 +10,84 @@ const Table = () => {
     const locationsData = await fetch('/locations')
       .then((res) => res.json())
       .then((data) => data.data);
-    setLocations(locationsData.map((location: any) => ({
+    const locs = locationsData.map((location: any) => ({
       id: location.id,
       name: location.name,
       address: location.address,
-    })));
+      ...(location.tag && { tag: location.tag }),
+    }));
+    setLocations(locs);
   };
 
   useEffect(() => {
     getExistingLocations();
   }, []);
 
   const addLocation = (newLocation: Location) => {
+    const body = { ...newLocation };
+    delete body.id;
```

Comment:
Where is this location getting an `id` that you'd have to delete. This request hasn't even been sent to the backend yet, so there shouldn't be an id. If you need to do this because of types, maybe just have `const {id, body} = {...newLocation}` and just don't use id? Might be a cleaner way, so do what you think is best. Just having a `delete` here doesn't feel right.

Reply:
Could also edit the Location type to have `id` as optional but that might cause other issues

Reply:
The `id` is there since the function takes in a `Location` from the form.

Nice. I didn't know that was an option. I'll use your method.